### DISCUSSION TITLE
Interface changing based on practice with remote access with low internet speed.

### DISCUSSION THREAD

**Discussion by VitaliyAT** [vladmandic/sdnext#1408] [Ideas]
Normally it based not only remote access. It's just logic grouping. Also, I know about sharing web interface but it temporally and forget all settings, plus it does not give you assess to disk - remote access better get through TeamViewer, AnyDesk, RustDesk or similar.

1) When you are Generate arts, main thing is "How match numbers should be generated?" Thats why it would be better move Batch count and batch size on place where styles right now. At the same time when you are using styles that menu grows down - it should be under the promts like long line - the same width as promts. 

As result all main options will be in first screen of first page - and you can start generating number of arts without scrolling down.
![image](https://github.com/vladmandic/automatic/assets/1639608/480f259f-6f97-4361-b5db-88474470540d)

2) Also when your internet access is low - picture is low too - and arrows in input fields is so small - Could you do it bigger - high as line of input field and place them not one on top of the other, but next to each other.  [input][up][down]

3) Could you add an option (check box) that do all digits from 0 to 1 to 0% to 100% ? It so annoying precisely click after dot or write each time [0.xxx]. Let it save in picture promt as 0.xxx , but in interface it will be more useful as %

**Comment by vladmandic** [comment]
> Also when your internet access is low - picture is low too - and arrows in input fields is so small - Could you do it bigger - high as line of input field and place them not one on top of the other, but next to each other. [input][up][down]

that is why you can create your own `user.css` and set sizes yourself

> Could you add an option (check box) that do all digits from 0 to 1 to 0% to 100% ? It so annoying precisely click after dot or write each time [0.xxx]. Let it save in picture promt as 0.xxx , but in interface it will be more useful as %

not with gradio as percentages are not supported as numbers.

**Reply by VitaliyAT** [reply]
Why % can't be just visual decoration? to comand line it will go as [0.xxx]. Just a visual adapter for human.

**Reply by vladmandic** [reply]
in any normal world, it could - but number input is a gradio component and styling those is a nightmare - you get what you get.

**Reply by VitaliyAT** [reply]
https://discuss.huggingface.co/t/percents-against-0-xxx/43016
Made discussion about % in Gradio workplace.
